recipes-dom0/dom0-image-thin-initramfs/files/meta-xt-prod-extra/recipes-bsp/u-boot/u-boot_%.bbappend

```diff
@@ -0,0 +1,8 @@
+FILESEXTRAPATHS_prepend := "${THISDIR}/files:"
+
+# this is the fake config, real one has to be defined
```

Comment:
this is not a fake config, this is a weak declaration...

Reply:
We need to be a bit more flexible here and be ready for a set of u-boot binaries for different domains,
e.g. for DomA/U, DomF etc.

Reply:
> this is not a fake config, this is a weak declaration...

resolved

Reply:
fixed in u-boot_%.bbappend , now, u-boot is installedin /xt/doma